src/commands/yubihsm/keys/generate.rs

```diff
@@ -119,6 +119,7 @@ impl Callable for GenerateCommand {
 impl_command!(GenerateCommand);
 
 /// Create an encrypted backup of this key under the given wrap key ID
+// TODO(tarcieri): unify this with the similar code in export?
```

Comment:
Would definitely be nice to do this before landing this PR as it's otherwise code duplication.